src/main/java/com/amazon/ion/impl/MarkerList.java

```diff
@@ -33,6 +33,13 @@ public int size() {
         return numberOfValues;
     }
 
+    /**
+     * @return The number of markers, including provisional markers, in the list.
+     */
+    public int provisionalSize() {
+        return provisionalIndex; // TODO Math.max(provisionalIndex, numberOfValues)? provisionalIndex + numberOfValues?
```

Comment:
Is this TODO still required?

Reply:
This is the only comment that needs to be addressed before I would approve the PR.

Reply:
Removed the TODO. When I wrote it, I wanted to double-check that no bytes could be read from the input between committing an element and adding a provisional element. I verified that's the case, but I had forgotten to remove the TODO.